raaghu-layouts/src/rds-comp-app-shell/rds-comp-app-shell.stories.tsx

```
@@ -248,15 +608,267 @@ export const Basic: Story = {
+              iconWidth: "20px",
+              label: "SemiDark",
+              val: "semiDark",
+            },
+          ]}
+          top_nav_logo="raaghu logo"
+          toggleItems={[]}
+          elementList={[]}
+          componentsList={[]}
+          languageLabel={""}
+          themeLabel={""}
+          onForgotPassword={function (isForgotPasswordClicked?: boolean): void {
```

ESLint

[error] Expected indentation of 20 spaces but found 10.

(indent)